tutorials/Additive_and_multiplicative_bias.ipynb

```diff
@@ -11,15 +11,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Bias tells us how well the mean forecast and mean observation correspond to each other. It can be used to tell us if there is an over or under-forecast tendency which can be used to inform us how a forecast system could easily be recalibrated.\n",
+    "Bias measures how well the mean forecast and mean observation correspond to each other. It can tell us whether there is an over or under-forecast tendency and inform how a forecast system could be easily recalibrated.\n",
```

Comment:
```suggestion
    "Bias measures how well the mean forecast and mean observation correspond to each other. It can tell us whether there is an over or under-forecast tendency and informs how a forecast system could be easily recalibrated.\n",
```